CHANGELOG.md

```diff
@@ -132,6 +132,10 @@
   JavaScript.
   ([Louis Pilfold](https://github.com/lpil))
 
+- Fixed a bug where calling a type or a constuctor `Else` would generate invalid
+  erlang code.
+  ([Giacomo Cavalieri](https://github.com/giacomocavalieri))
```

Comment:
Could you merge this with the `maybe` note above please 🙏

Reply:
Ok!